systemd/volumes.go

```diff
@@ -208,3 +208,8 @@ func chown(name, uid, gid string) error {
 }
 
 const dirPerms = 02750
+
+func cleanPodEphemeralVolumes(podId string) error {
```

Comment:
as a nit: use pod.ObjectMeta.UID as the parameter here and stringify in the function here?